pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Description
```

Comment:
Maybe use ## to be the same as the "Type of Change" section below?

Reply:
Make sense! I will fix that!